test/integration/connect/envoy/helpers.windows.bash

```diff
@@ -1,5 +1,32 @@
 #!/bin/bash
 
+function check_hostport {
```

Comment:
In my opinion we can use a map to identity hosts/ports to replace. On this way, any changes should be easier and improve the readability of the function/script

Eg. 
```
declare -A hostportsMap=(
    ["localhost:8500"]="envoy_consul-primary_1:8500"
    ["localhost:21000"]="envoy_s1-sidecar-proxy_1:21000"
    ["localhost:21001"]="envoy_s2-sidecar-proxy_1:21000"
    ["localhost:19000"]="envoy_s1-sidecar-proxy_1:19000"
    ["localhost:19001"]="envoy_s2-sidecar-proxy_1:19000"
    ["localhost:19000/stats"]="envoy_s1-sidecar-proxy_1:19000/stats"
    ["localhost:19001/stats"]="envoy_s2-sidecar-proxy_1:19000/stats"
)

function check_hostport {
    if [[ -v "hostportsMap[$1]" ]] ; then
        echo ${hostportsMap[$1]}
        return
    fi

    return 1
}
```